BotRoyaleUnity/Assets/Scripts/Networking/SocketConnectionHandler.cs

```diff
@@ -126,10 +126,40 @@ public void EmitGameMessage(JSONObject data, Action<string> onMessageSent = null
         }
         else
         {
-            socket.Emit("game-message", data.ToString(), onMessageSent); ;
+            socket.Emit("game-message", data.ToString(), onMessageSent);
         }
     }
 
+    public void EmitCurrentParts(GameObject robot){
+        Vector2Int centerPos = robot.GetComponent<PartHandler>().GetCenterPos();
```

Comment:
I think this code would make more sense in the PartHandler class, since it depends on having a reference to it and many of it's variables